be/test/CMakeLists.txt

```diff
@@ -295,129 +411,21 @@ set(EXEC_FILES
-        ./util/coding_test.cpp
-        ./util/core_local_test.cpp
-        ./util/countdown_latch_test.cpp
-        ./util/crc32c_test.cpp
-        ./util/dynamic_cache_test.cpp
-        ./util/exception_stack_test.cpp
-        ./util/fail_point_test.cpp
-        ./util/faststring_test.cpp
-        ./util/file_util_test.cpp
-        ./util/filesystem_util_test.cpp
-        ./util/frame_of_reference_coding_test.cpp
-        ./util/json_util_test.cpp
-        ./util/md5_test.cpp
-        ./util/monotime_test.cpp
-        ./util/mysql_row_buffer_test.cpp
-        ./util/new_metrics_test.cpp
-        ./util/parse_util_test.cpp
-        ./util/path_trie_test.cpp
-        ./util/path_util_test.cpp
-        ./util/priority_queue_test.cpp
-        ./util/rle_encoding_test.cpp
-        ./util/runtime_profile_test.cpp
-        ./util/scoped_cleanup_test.cpp
-        ./util/string_parser_test.cpp
-        ./util/string_util_test.cpp
-        ./util/tdigest_test.cpp
-        ./util/thread_test.cpp
-        ./util/trace_test.cpp
-        ./util/uid_util_test.cpp
-        ./util/utf8_check_test.cpp
-        ./util/int96_test.cpp
-        ./util/bit_packing_test.cpp
-        ./util/gc_helper_test.cpp
-        ./util/lru_cache_test.cpp
-        ./util/arrow/starrocks_column_to_arrow_test.cpp
-        ./util/starrocks_metrics_test.cpp
-        ./util/system_metrics_test.cpp
-        ./util/ratelimit_test.cpp
-        ./util/cpu_usage_info_test.cpp
-        ./util/timezone_utils_test.cpp
-        ./util/concurrent_limiter_test.cpp
-        ./util/stack_trace_mutex_test.cpp
-        ./gutil/cpu_test.cc
-        ./gutil/sysinfo-test.cc
-        ./service/lake_service_test.cpp
         ./storage/compaction_parallelization_test.cpp
         ./storage/lake/persistent_index_memtable_test.cpp
         ./storage/lake/lake_persistent_index_test.cpp
         )
 
-if ("${WITH_STARCACHE}" STREQUAL "ON" OR "${WITH_CACHELIB}" STREQUAL "ON")
-    list(APPEND EXEC_FILES ./block_cache/block_cache_test.cpp)
-    list(APPEND EXEC_FILES ./io/cache_input_stream_test.cpp)
-endif ()
-
-if ("${USE_STAROS}" STREQUAL "ON")
-    list(APPEND EXEC_FILES ./fs/fs_starlet_test.cpp)
-    list(APPEND EXEC_FILES ./storage/lake/local_pk_index_manager_test.cpp)
-endif ()
-
-if (USE_AVX2)
-    set(EXEC_FILES ${EXEC_FILES} ./column/avx_numeric_column_test.cpp)
-endif ()
-
 add_library(starrocks_test_objs OBJECT ${EXEC_FILES})
 SET_TARGET_PROPERTIES(starrocks_test_objs PROPERTIES COMPILE_FLAGS "-fno-access-control")
 add_executable(starrocks_test test_main.cpp $<TARGET_OBJECTS:starrocks_test_objs>)
 
+add_library(starrocks_storage_test_objs OBJECT ${STORAGE_EXEC_FILES})
+SET_TARGET_PROPERTIES(starrocks_storage_test_objs PROPERTIES COMPILE_FLAGS "-fno-access-control")
+add_executable(starrocks_storage_test test_main.cpp $<TARGET_OBJECTS:starrocks_storage_test_objs>)
+
 TARGET_LINK_LIBRARIES(starrocks_test ${TEST_LINK_LIBS})
+TARGET_LINK_LIBRARIES(starrocks_storage_test ${TEST_LINK_LIBS})
```

Comment:
it is not really storage related testing, maybe we can name it starrocks_test_part1, starrocks_test_part2 ..., and also add comments here to let others know why it is doing like this here.

Reply:
Done, PTAL.